### ISSUE TITLE
Per-system ban/kick option

### ISSUE THREAD

**Issue by Nerus87** [L-EARN/NoCheatZ-4#146]
### The name of the game your server is running on and the version of the game.
Counter-Strike: Source

### The version of the plugin release (R) (Otherwise, link to commit/pull request/fork). If using a custom build, please list your compiler version, your server and builder OS versions.
v4-beta-r5 (94dd222)

### Expected behavior?
Kick player does not ban as preventing to use Auto Attack.

### Current behavior? (You can link a demo/video and logfiles. Don't forget to strip personal and sensitive information before.)
Player banned.

### Steps to reproduce the problem?
Use mause with double clieck button.

**Comment by L-EARN** [comment]
Hi,

Can I have a copy of your cfg file ?
Or faster : Have you set AllowWriteBans to No ?

Just so I know if this is a new bug or a silent one I wasn't aware of ...

Thank you

**Comment by Nerus87** [comment]
**Can I have a copy of your cfg file ?**

// Testers

ncz ConCommandTester enable
ncz ConVarTester enable
ncz EyeAnglesTester enable
ncz JumpTester disable
ncz AutoAttackTester disable
ncz SpamChangeNameTester enable
ncz SpamConnectTester enable
ncz SpeedTester enable

// Blockers

ncz AntiFlashbangBlocker enable
ncz RadarHackBlocker enable
ncz AntiSmokeBlocker enable
ncz WallhackBlocker enable
ncz BhopBlocker enable

// Misc

ncz BanRequest CanKick Yes
ncz BanRequest CanBan Yes
ncz BanRequest **AllowWriteBans Yes**
ncz Logger AllowChat off
ncz Logger AlwaysFlush off

Basically, I want to ban for all of the others detections, but only in AutoAttackTester case will be nice if I can kick the player for it.

**Comment by L-EARN** [comment]
aaahhhhh ... ok.

So this is a feature request, not a bug.

If I understand, you want to be able to set a system allowed to ban/kick independently. Am I right ?

**Comment by Nerus87** [comment]
Yes, it will be nice if we will have chosen behavior of the selected system.

**Comment by L-EARN** [comment]
Besides that, is there anything not working good with JumpTester, AutoAttackTester and ConVarTester ?

**Comment by Nerus87** [comment]
- JumpTester - players don't like kick or ban on it, we just use BhopBlocker to prevent bunny hops and is ok.
- ConVarTester contains blocking some dangerous convars functionalities it is good, but will be nice if we have some access to some convars but only if an admin can execute them, like in "net_graph 3" case e.g. for server and network performance diagnostic (list of convars and list of admins).

**Comment by L-EARN** [comment]
Implemented now on master branch.
I need to perform testings and if everything is good I'll roll out a release.
